test/Dockerfile

```diff
@@ -28,4 +36,6 @@ WORKDIR $GOPATH/src/github.com/skroutz/rafka
 
 EXPOSE 6380
 
+RUN echo /usr/local/lib >> /etc/ld.so.conf.d/x86_64-linux-gnu.conf && ldconfig
```

Comment:
Maybe we can avoid yet another image layer here, i think that you can use `LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib` as an argument in CMD.

Reply:
I've done it by `./configure --libdir=.....`